Sources/SwiftKuery/QueryBuilder.swift

```diff
@@ -47,20 +47,30 @@ public class QueryBuilder {
         case namesCount
     }
     
+    /// Database types that could be used for building a query 
+    /// and supported by `Swift-Kuery`
+    public enum Database: String {
+        case unknown
+        case postgreSQL
+        case sqLite
+    }
+    
```

Comment:
Could you please change this part, and replace it with two boolean variables: withDeleteRequiresUsing and withUpdateRequiresFrom similar to the vars below.